legate/driver/args.py

```diff
@@ -41,58 +40,7 @@
 __all__ = ("parser",)
 
 
```

Comment:
Should we also remove `parse_library_command_args` from `util/args.py`?

Reply:
ah yes we certainly could

Reply:
[7d2e5b4](https://github.com/nv-legate/legate.core/pull/572/commits/7d2e5b410481c9e421d4f364653601d7daa482f8)